src/NServiceBus.Core/obsoletes-v8.cs

```diff
@@ -200,6 +200,7 @@ public interface IConfigureComponents
         void ConfigureComponent(Type concreteComponent, DependencyLifecycle dependencyLifecycle);
 
         [ObsoleteEx(
+            Message = "Ensure to explicitly register all interfaces of a specific type as this will no longer be done automatically.",
```

Comment:
This is not entirely true. the IServiceCollection.ConfigureComponent extension method (https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/ObjectBuilder/ServiceCollectionExtensions.cs#L31) does continue to register interfaces and replicates the existing behavior. Only moving away from the `ConfigureComponents` API entirely requires the described changes.

Reply:
this is also described in the upgrade guide here: https://docs.particular.net/nservicebus/upgrades/7to8/dependency-injection#registercomponents-changes but it seems there should be more explicit guidance to move away from ConfigureComponent

Reply:
@timbussmann I don't understand your comment. Could you propose a different help text? Please see the PR description for what this should prevent.

Reply:
@ramonsmits from what you describe, you want to add the suggested message to these extension points: https://github.com/Particular/NServiceBus/blob/master/src/NServiceBus.Core/ObjectBuilder/ServiceCollectionExtensions.cs#L15

Reply:
@timbussmann I guess those should have the message too. Extended the message to contain the same guidance.

Reply:
👍 for the additional changes, I still think that we shouldn't add the message on this extension method because it will be the first thing on the obsolete message but then the obsolete message will also point to the Configure extension method that still does register interfaces so it becomes kind of confusing.

Reply:
@timbussmann I don't understand your comment about this method. The obsolete is on `ConfigureComponent`. There is no other method to apply this to as this is the obsoleted method.